src/lib/core/ripple/ripple.spec.ts

```diff
@@ -180,6 +180,47 @@ describe('MdRipple', () => {
       expect(pxStringToFloat(ripple.style.width)).toBeCloseTo(2 * expectedRadius, 1);
       expect(pxStringToFloat(ripple.style.height)).toBeCloseTo(2 * expectedRadius, 1);
     });
+
+    it('create ripple with correct position when page is scrolled', () => {
+      let elementTop = 600;
+      let elementLeft = 750;
+      let pageScrollTop = 500;
+      let pageScrollLeft = 500;
+      let left = 50;
+      let top = 75;
+
+      // Add a very large element to make the page scroll
+      let veryLargeElement: HTMLElement = document.createElement('div');
+      veryLargeElement.style.width = '4000px';
+      veryLargeElement.style.height = '4000px';
+      document.body.appendChild(veryLargeElement);
+      document.body.scrollTop = pageScrollTop;
+      document.body.scrollLeft = pageScrollLeft;
+
+      rippleElement.style.left = `${elementLeft}px`;
+      rippleElement.style.top = `${elementTop}px`;
```

Comment:
* Setting the `left` and `top` of the rippleElement probably won't do what you want if the element isn't `position: absolute`. 
* I'm not sure if the rippleElement is in the DOM; it would have to be for the positioning to be right.

This might be why the tests are failing.